### PR TITLE
chainloop-cli 0.20.0 (new formula)

### PR DESCRIPTION
**Pull request by p-linnane**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/c/chainloop-cli.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X github.com/chainloop-dev/chainloop/app/cli/cmd.Version=#{version}
+    ]
+
+    system "go", "build", *std_go_args(output: bin/"chainloop", ldflags: ldflags), "./app/cli"
+
+    generate_completions_from_executable(bin/"chainloop", "completion", base_name: "chainloop")
```
Comment (chenrui333) [inline Formula/c/chainloop-cli.rb:19]:
```suggestion
    generate_completions_from_executable(bin/"chainloop", "completion")
```

File: Formula/c/chainloop-cli.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+class ChainloopCli < Formula
```
Comment (chenrui333) [inline Formula/c/chainloop-cli.rb:1]: why not just Chainloop rather than ChainloopCli?

Comment (p-linnane) [reply Formula/c/chainloop-cli.rb:1]: You use the CLI to interact with the Chainloop platform. I modeled this after the `lacework-cli` formula, which is the same sort of situation.

Comment (chenrui333) [reply Formula/c/chainloop-cli.rb:1]: maybe lacework-cli should be lacework formula as well, my initial thought was there would be some naming conflict. 

most of the formulae are CLI, so there is no need to specify `-cli` IMO

Comment (p-linnane) [reply Formula/c/chainloop-cli.rb:1]: I think it makes sense to specify CLI for formulae that are just CLI to interact with platforms. It reduces ambiguity. I don't see us renaming things like `awscli` either..

**Review by chenrui333 — APPROVED** [review]

File: Formula/c/chainloop-cli.rb (hunk 2 above)
Comment (chenrui333) [reply Formula/c/chainloop-cli.rb:1]: yeah, I guess it is fine.

Comment (chenrui333) [reply Formula/c/chainloop-cli.rb:1]: personally, I still prefer chainloop over chainloop-cli, but nothing blocking merging this PR though

**Pull request merged**